Cargo.toml

```diff
@@ -671,3 +671,10 @@ field_names = "0.2.0"
 semver = "1.0.16"
 spinners = "4.1.0"
 include_dir = "0.7.3"
+
+# Patches for revm, needed by native bridge
```

Comment:
Could you add a TODO and comment here on when we don't need the patch anymore?

Reply:
we need patches here because forge requires them. I can add explanation for context